build.sbt

```diff
@@ -43,14 +43,9 @@ sparkComponents ++= Seq("graphx", "sql", "mllib")
 // add any Spark Package dependencies using spDependencies.
 // e.g. spDependencies += "databricks/spark-avro:0.1"
 
-libraryDependencies += "org.scalatest" %% "scalatest" % defaultScalaTestVer % "test"
+libraryDependencies += "org.slf4j" % "slf4j-api" % "1.7.21"
```

Comment:
is there a reason for .21?
spark is running with 1.7.16
https://github.com/apache/spark/blob/9b6baeb7b9f73e1a38581f481ea7232db712deb8/pom.xml#L119

Reply:
@felixcheung thanks, I totally missed that part, which is checking which version Spark is running. I've just picked up one the version which I've always used